convention-develocity-gradle-plugin/README.md

```diff
@@ -66,7 +66,7 @@ cd plugins/gradle-5-or-newer
 ./gradlew publishToMavenLocal
 ```
 
-Once you have published the plugins, you can run the four example builds under `examples`:
+Once you have published the plugins, you can run the five example builds under `examples`:
```

Comment:
Might be easier just to say `run the example builds` so these don't go out of sync.

Reply:
Good point.

Reply:
Fixed in 058f77ec751600a6cea0438ee6825b5121acf44c